assets/cases/TestList/backbutton.ts

```diff
@@ -67,6 +67,11 @@ export class BackButton extends Component {
                     continue;
                 }
             }
+
+            if (sys.platform === sys.Platform.XIAOMI_QUICK_GAME && str.endsWith('lod.scene')) {

```

Comment:
Please add the reason to do it like this.

Reply:
Done